src/libYARP_OS/src/Network.cpp

```diff
@@ -845,6 +845,10 @@ void NetworkBase::initMinimum(yarp::os::yarpClockType clockType, yarp::os::Clock
     if (__yarp_is_initialized == 0) {
         // Broken pipes need to be dealt with through other means
         yarp::os::impl::signal(SIGPIPE, SIG_IGN);
+        // Multinamespace is a c++11 singleton and need to be initialized
+        // before the first port that is opened and it has to exist until
+        // the last port is closed.
+        getNameSpace();
```

Comment:
I'm not sure where it used to be, but I believe that this should be *after* ACE and (I think) after all the other initialization (quiet, verbose and probably turboboost)